src/game/stage/camera.c

```diff
@@ -1502,6 +1506,11 @@ void CreateStageBg_Zone6_Boss(void)
     gBgScrollRegs[0][1] = 0;
     gBgScrollRegs[3][0] = 0;
     gBgScrollRegs[3][1] = 0;
+#if TAS_TESTING
+    // This has an effect on the RNG.
```

Comment:
I feel like this should just be set the same for all non matching builds. Otherwise I feel like this could affect game play for someone who is used to speed running in a certain way. Why not just change the above `if (IS_MULTI_PLAYER) {` in `CreateStageBg_Zone6_Acts` to always be true for portable builds?

Reply:
Very good idea!